Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "redjubjub"
-edition = "2018"
+edition = "2021"
```

Comment:
This change does not change the MSRV, as `reddsa 0.5.0` already requires `1.65`.